core/app/models/concerns/spree/active_storage_adapter/attachment.rb

```diff
@@ -19,11 +19,11 @@ def exists?
       end
 
       def filename
-        blob.filename.to_s
+        blob&.filename.to_s
```

Comment:
Should this have two safe operators or are you fine with the empty string when `blob` is nil?

```suggestion
        blob&.filename&.to_s
``

Reply:
I'm not sure it is needed, the safe navigation was there to avoid calling `Blob#filename` on `nil`, returning `nil` or an empty string should not make a difference to `Attachement#filename` users